smoke-tests/apps/RateLimitedSampling/src/main/java/com/microsoft/applicationinsights/smoketestapp/HealthCheckServlet.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
per our earlier discussion, we will use index.js over this for consistency with the rest of the smoke tests?

Reply:
I'm not sure, I don't really like web.xml, I prefer annotation based, but I created a work item to make them all consistent one way or another